### ISSUE TITLE
What are you using Engo for?

### ISSUE THREAD

**Issue by paked** [EngoEngine/engo#522]
Hey everyone!

I've noticed recently that there are more and more people who are not maintainers commenting on threads (definitely a good thing!), which raises the question in the title: What are you using Engo for?

A brief description or even some (shudder) screenshots would be awesome. If you have any feature requests, or things we could add to make getting used to the framework easier you can chuck them in a post here too (someone might spin them off into an issue eventually).

**Comment by Noofbiz** [comment]
I was actually thinking about trying to host a game jam for `engo` on something like https://itch.io/jams once we hit the 1.0 milestone (or maybe before, like have a 0.9 alpha version) and collect people's feature requests as well as some games to feature on the site from that. It could also be a good way to get people interested in `engo` who may otherwise overlook it.

**Comment by RaindeerMokus** [comment]
What an old issue and basicly no comments here, I'm suprised. I've just found  engo on the weekend, so I just done the tutorial, and looked through the demos and started to look through the code to know most of the things built into it, and so far a really like it. I'm planning to do a sidescroller woth this, as a first project. To be exact, I want to make a sidescroller based on and led by my poems. I mostly have the ideas for it, and I've started to work on the graphics, which is, well, for sure not my strongest skill. I'll sure keep you updated about how things are going on.
The jam also sounds amazing, are there amy news about that?

**Comment by Noofbiz** [comment]
Yeah, an issue is hard to get people to comment on 😜 If you're not watching the repo there's no way to know if an issue exists.

I think test coverage and everything looks good. I'd like to get TravisCI to also run gopherjs tests, and I think I'll move the render system improvements to v1.1, so then we should be good to go with v1.0. The game jam should follow. I think I'll make it for the month of June just because June first sounds like as good a time as any to start it. Once v1.0 is out, I'll do a freeze on updating engo to work on the tutorial series and add more concepts to the site.
